eng/Versions.props

```diff
@@ -117,7 +117,7 @@
     <MicrosoftVisualStudioTextUIPackageVersion>16.0.177-g0ae5fa46a1</MicrosoftVisualStudioTextUIPackageVersion>
     <MicrosoftVisualStudioThreadingPackageVersion>16.0.102</MicrosoftVisualStudioThreadingPackageVersion>
     <MonoAddinsPackageVersion>1.3.8</MonoAddinsPackageVersion>
-    <MonoDevelopSdkPackageVersion>1.0.1</MonoDevelopSdkPackageVersion>
+    <MonoDevelopSdkPackageVersion>1.0.10</MonoDevelopSdkPackageVersion>
```

Comment:
@Pilchie I assume we don't need a process to update our Mono SDK dependency. Correct?

Reply:
Correct - we should be fine here.